syntax = "proto3";

package AMMSgRPC_proto;

service RealTimeModelServing{
  rpc DetectDefect (ImageDataPacket) returns (PredictResult);
}

message ImageDataPacket
{
	string Name = 1;
	int32 LayerNum = 2;
	bytes Datas = 3;
}

message PredictResult {
    string ClassResult = 1;
    int32 ClassReusltPercent = 2;
}